sdcm/utils/functions.py

```diff
@@ -0,0 +1,37 @@
+# This program is free software; you can redistribute it and/or modify
```

Comment:
Module named function ? those are function to parse stress commands

Why not put those functions on the stress classes ? so each thread could compute the duration/timeout ?